### DISCUSSION TITLE
I'm looking for something like this but with real-time parsing. Any ideas?

### DISCUSSION THREAD

**Discussion by gibson-kim** [Tyrrrz/DiscordChatExporter#760] [Q&A]
Just want something to scan recent chat data in real-time and notify me if certain keywords or messages by certain users come up. Is there anything like this available?

**Comment by 96-LB** [comment]
The best tool for this is to use a bot account. I'm not aware of any public bots in specific that can do this, but it's possible that these features are included as part of some bot's moderation toolkit. If you have coding knowledge, it's a pretty simple feature to create.
